changelog.d/10036.misc

```diff
@@ -0,0 +1 @@
+Don't hammer the database for destination retry timings every ~5mins.
```

Comment:
Maybe?

```suggestion
Properly invalidate caches for destination retry timings every (instead of expiring entries every 5 minutes).
```

Although still not descriptive to understand if it will fix anything. 🤷